### DISCUSSION TITLE
"Transcribe" function has arguments: multilingual, output_language

### DISCUSSION THREAD

**Discussion by toanhuynhnguyen** [SYSTRAN/faster-whisper#1045] [Q&A]
"Transcribe" function has arguments: multilingual, output_language. How can I use these arguments to transcribe multilingual audio?
![image](https://github.com/user-attachments/assets/6de059e2-ccc0-48bb-9ba5-de81d74659c2)

**Comment by kailust** [comment]
ERROR:__main__:Error during transcription: transcribe() got an unexpected keyword argument 'multilingual'
ERROR:__main__:Transcription failed with error: transcribe() got an unexpected keyword argument 'multilingual'
when I did this
segments, info = model.transcribe(path, beam_size=5, word_timestamps=False, multilingual=True, output_language="hybrid")

**Reply by toanhuynhnguyen** [reply]
I think you also specify the "language" argument.
